tools/topology/topology2/cavs-nocodec.conf

```diff
@@ -12,6 +12,7 @@
 <virtual.conf>
 <passthrough-capture.conf>
 <passthrough-be.conf>
+<wov-detect.conf>
```

Comment:
Where are you enabling wov ins sof-mtl-nocodec? Can what you really mean is Add INCLUDE_WOV in CMAKELists.txt when the feature is ready?

Reply:
No, I didn't enable it. Need a patch from Chao for hw_params update. Otherwise it will block CI test.

Reply:
also fix the commit message and please remove all references to DUMMY in the PR

Reply:
OK